manual/Website1/src/App.jsx

```diff
@@ -18,7 +18,7 @@ class App extends Component {
   componentDidMount() {
     corsImport('http://localhost:3002/importManifest.js').then(() => {
       this.setState({ manifestLoaded: true });
-      importDependenciesOf('http://localhost:3002', 'website-two', 'TitleComponent').then((url) => {
+      importDependenciesOf('http://localhost:3002/', 'website-two', 'TitleComponent').then((url) => {
```

Comment:
@seanvern these utility functions need a more reliable way to stitch together a url, sometimes theres a missing `/` or two `//`